lib/mollie/client.rb

```diff
@@ -60,5 +60,14 @@ def refund_payment(payment_id)
       JSON.parse(response.body)
     end
 
+    def methods(method = nil)
+      response = self.class.get("/methods/#{method}",
+        :headers => {
+          'Authorization' => auth_token
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.